modular_zubbers/master_files/code/modules/entombed_quirk/code/entombed_mod.dm

```diff
@@ -0,0 +1,95 @@
+/datum/mod_theme/entombed
+	name = "fused"
+	desc = "Circumstances have rendered this protective suit into someone's second skin. Literally."
+	extended_desc = "Some great aspect of someone's past has permanently bound them to this device, for better or worse."
+
+	default_skin = "civilian"
+	armor_type = /datum/armor/mod_entombed
+	resistance_flags = FIRE_PROOF | ACID_PROOF // It is better to die for the Emperor than live for yourself.
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	siemens_coefficient = 0
+	complexity_max = DEFAULT_MAX_COMPLEXITY - 5
+	charge_drain = DEFAULT_CHARGE_DRAIN
+	slowdown_inactive = 2.5 // very slow because the quirk infers you rely on this to move/exist
+	slowdown_active = 0.95
+	inbuilt_modules = list(
+		/obj/item/mod/module/joint_torsion/entombed,
+		/obj/item/mod/module/storage,
+	)
+	allowed_suit_storage = list(
+		/obj/item/tank/internals,
+		/obj/item/flashlight,
+	)
+
+/datum/armor/mod_entombed
+	melee = ARMOR_LEVEL_WEAK
```

Comment:
Use values, not these defines. These defines only exist to make it harder to read the code